swin_transformer/requirements.txt

```
@@ -0,0 +1,2 @@
+torch
```

```suggestion
torch>=2.6
```

Due to usage of `torch.accelerator`.